Client/src/Components/CustomThreshold/index.jsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Stack, Typography } from "@mui/material";
+import Field from "../Inputs/Field";
+import Checkbox from "../Inputs/Checkbox";
+import { useTheme } from "@emotion/react";
+import PropTypes from "prop-types";
+
+export const CustomThreshold = ({
```

Comment:
Be sure to include JSdoc for all functions/components before opening the final pull request :+1: